pkg/service/labels/labels.go

```diff
@@ -21,9 +21,18 @@ func merge(maps ...map[string]*string) map[string]*string {
 }
 
 func GetNodeLabel(nodeSpec *proto.NodeSpec) map[string]*string {
+
+	instance := ""
+	if nodeSpec.MachineType != "" {
+		instance = nodeSpec.MachineType
+	}
+	if nodeSpec.Instance != "" {
+		instance = nodeSpec.Instance
+	}
```

Comment:
This is done in a lot of places, can we just add a utility function for this in common pkg?

Reply:
we cant put this in there, would mean common needs to know nodeSpec, not ideal and also creates cyclic.
in other case having  it together serves different purpose in readabilty.

Reply:
We should pass MachineType and Instance strings from nodeSpec to the util func.

Reply:
it would complicate for different scenarios. This change is only for labels.
I don see value in adding util functions for this. Can revisit later.

Reply:
Not just labels we do the same thing in azure and aws node.go

Reply:
they are actually using different istance names, we are adding the same to label.
We will add machine type such as `m`, `l`, etc in tags where as actual instance are used for azure and aws cluster nodes

Reply:
Why so? Why not use the actual instances names in tags?